### DISCUSSION TITLE
Problem with Report Generation from ufdr

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#868] [Q&A]
I'm trying to generate a digital report (iped 3.18.9) from bookmarks containing audio and html chat files.

Inserting only the audio files (.opus), the report is generated, but without a link to the files.

_Worker-1 Error creating temp file xxxxxx.ufdr/iPhone XXXXXX/mobile/Containers/Shared/AppGroup/group.net.whatsapp.WhatsApp.shared/Message/Media/xxxxxxxxx@s.whatsapp.net/1/f/1f48fceb- a358-440f-9ac2-ea19bf924bee.opus java.io.IOException: net.lingala.zip4j.exception.ZipException: zip file does not exist
2021-12-07 15:07:04 [ERROR] [indexer.parsers.RegistryParser] Error testing rip (RegRipper): registry reports will NOT be created. Command line: E:\xxxx\xxxxx\iped\indexer/../regripper/rip Check if perl module 'Parse::Win32Registry' is installed.
2021-12-07 15:07:04 [ERROR] [indexer.parsers.OutlookDBXParser] DBX parser not found, DBX mailboxes will NOT be expanded._

The path exists indexed in iped, as well as I can export the file individually.

When I include the chat file, the process of generating the report gives an error.

Any suggestion?

**Comment by lfcnassif** [comment]
Could you share the full log? You can replace your sensitive strings by similar ones.

**Comment by gisms** [comment]
I'll do some more tests and return with the logs.

**Comment by gisms** [comment]
I was able to better reproduce the problem. In fact, it occurs when the iped loses the reference to the image file (reorganization of directories), for the case it was a ufdr file. Passing the new path of the image to Iped when opening the tool was not enough for the correct generation of the report.
Sometimes iped requests the image during report generation, but it still doesn't generate everything correctly.

If more people could test to check it would be interesting

**Reply by lfcnassif** [reply]
That's important information, thanks. I can try to take a look after I return from vaccation in 10 days, if no one else takes a look at this before me.

**Reply by lfcnassif** [reply]
Returning from vacation. I was able to reproduce this on 3.18.11 after moving an ufdr evidence to a different folder:

> Inserting only the audio files (.opus), the report is generated, but without a link to the files.

But not this:

> When I include the chat file, the process of generating the report gives an error.

Neither this:

> Sometimes iped requests the image during report generation

I'll take a closer look at the issue that I was able to reproduce. Please try to reproduce the others on 3.18.11 and please provide further details to reproduce them.

**Reply by lfcnassif** [reply]
@gisms I opened #880 and hopefully fixed it. Check if this snapshot works for you: https://github.com/sepinf-inc/IPED/actions/runs/1603609605

> Sometimes iped requests the image during report generation, but it still doesn't generate everything correctly.

This happens if you don't select the new evidence location before creating the report, it worked for me with the referenced snapshot.

**Reply by gisms** [reply]
I've tested what you asked for, actually since version 10 this error of not generating the report has stopped. Just this question of the links to the audios. I will teste the snapshot.

**Reply by lfcnassif** [reply]
Great, thank you for reporting this and for testing!
